M2/Macaulay2/packages/NumericalImplicitization.m2

```diff
@@ -64,9 +65,16 @@ newPackage("NumericalImplicitization",
         "generalCombinations",
         "imageSlice",
         "witnessPointPairs",
-	"isOnImage"
+	"isOnImage",
+        "realPoint",
+        "nelderMead",

```

Comment:
"nelderMead" is a clumsy name.  What about "optimize" instead?  It's descriptive.

Reply:
I'm a little hesitant to refer to this method by a totally generic name, since the function is exactly implementing the Nelder-Mead algorithm. Calling it "optimize" might conflict with other optimization methods added in the future. (Also, I was not really intending for users to call this function directly, and only exposed it via the user-facing, documented method `realPoint`.)

Reply:
If you add other optimization methods in the future, the choice can be offered to the user like this:
```m2
optimize(..., Strategy=>NelderMead)
```
, with one of them being the default.  It's better if function names are verbs describing what is to be done.  The choice of algorithm can be described in the documentation.

Reply:
Hi Dan, I'm still not entirely sure about this name change. I am perfectly happy with the possibility of a (future) general optimization function to call this as a strategy (or even a hook), but I don't know if this function should play that role. Would you find it acceptable to just not export this function (so it becomes an internal function)? Alternatively, what about something like "optimizeNelderMead"?

Reply:
Okay.  That's a bit prettier, because both of the names get capitalized, and there is something descriptive.

Reply:
OK, great! Marking as resolved.